x-pack/plugins/secops/public/pages/home/index.tsx

```diff
@@ -132,7 +132,7 @@ const HomePageComponent = pure<Props>(({ theme }) => (
                     <Switch>
                       <Redirect from="/" exact={true} to="/overview" />
                       <Route path="/overview" component={Overview} />
-                      <Route path="/hosts" component={Hosts} />
+                      <Route path="/hosts" component={HostsContainer} />
```

Comment:
Is this fine for naming conventions? Or would the preference be to keep this as `Hosts` and refactor the default hosts page to be `HostsDefault` or something similar?